.vscode/launch.json

```diff
@@ -307,12 +336,62 @@
         "${workspaceRoot}/packages/salesforcedx-sobjects-faux-generator/out/test/**/*.js"
       ],
       "preLaunchTask": "Compile"
+    },
+    {
+      "name": "Launch Replay Debugger adapter",
+      "type": "node",
+      "request": "launch",
+      "cwd": "${workspaceRoot}",
+      "program": "${workspaceRoot}/packages/salesforcedx-vscode-replay-debugger/out/src/adapter/apexReplayDebug.js",
+      "args": [
+        "--server=4711"
```

Comment:
Is there a reason why we are choosing the same port, i.e, 4711 for both the Apex Debugger and the Apex Replay Debugger? This makes it harder to be able to debug both of them at the same time, right?